packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { randomId } from '@rentpath/react-ui-utils'
+import Star from './SampleSvgs/Star'
+
+export default class RatingBar extends Component {
+  static propTypes = {
+    theme: PropTypes.object,
+    ratingScale: PropTypes.number,
+    rating: PropTypes.number,
+    total: PropTypes.string,
+    RatingItem: PropTypes.node,
+  }
+  static defaultProps = {
+    theme: {},
+    RatingItem: Star,
+  }
+
+  get ratingItems() {
+    const {
+      ratingScale,
+      rating,
+      RatingItem,
+      ...props
+    } = this.props
+
+    return [...Array(ratingScale).keys()].map(index => (
+      <RatingItem
+        key={randomId('rating')}
+        id={randomId('rating-item')}
```

Comment:
What are your thoughts on moving the `id` generation to the `Star` component though?